src/templates/invoice/Invoice.tsx

```diff
@@ -0,0 +1,160 @@
+import { css } from "@emotion/core";
+import { TemplateProps } from "@govtechsg/decentralized-renderer-react-components";
+import React, { FunctionComponent } from "react";
+import { Invoice } from "./types";
+import { format } from "date-fns";
+
+const container = css`
+  font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
+  color: #4e4e50;
+  width: 100%;
+  height: 100vh;
+  max-height: 1000px;
+  max-width: 980px;
+  position: relative;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+`;
+
+export const InvoiceTemplate: FunctionComponent<TemplateProps<Invoice> & { className?: string }> = ({
+  document,
+  className = ""
+}) => {
+  const { id, date, customerId, terms, billFrom, billTo, invoiceItems } = document;
+
+  return (
+    <div>
+      <div css={container} className={className}>
+        <h1 style={{ color: "#4172AF", textAlign: "right" }}>INVOICE</h1>
+        <div className="m-0" style={{ position: "absolute", top: "0" }}>
+          <h1 style={{ color: "#4172AF", textAlign: "left" }}>{billFrom?.companyName}</h1>
+          <h3 style={{ textAlign: "left" }}>{billFrom?.streetAddress}</h3>
+          <h3 style={{ textAlign: "left" }}>
```

Comment:
want to try using emotion styled components rather than repeating inline styles?